public/requests.js

```diff
@@ -0,0 +1,134 @@
+import md5_encode from './MD5.js'
+
+/**
+ * Функция добавления eventListener на кнопку поиска трека
+ */
+(function() {
+    let search_input = document.querySelector('.header__search');
```

Comment:
переменные, которые не меняются, лучше делать const
```suggestion
    const search_input = document.querySelector('.header__search');
```